Livecheckables/markdown.rb

```diff
@@ -0,0 +1,6 @@
+class Markdown
+  livecheck do
+    url :homepage
+    regex(/href=.*?Markdown_v?(\d+(?:\.\d+)+)\.zip/i)
```

Comment:
```suggestion
    regex(/href=.*?Markdown.v?(\d+(?:\.\d+)+)\.zip/i)
```

It may be better to match anything here instead of `_`, so this would still match the archive if it was named `Markdown-1.2.3.zip`, for instance.